src/Native/System.Security.Cryptography.Native/pal_ssl_extra_cert.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_ssl_extra_cert.h"
+
+extern "C" int AddExtraChainCertToSslCtx(SSL_CTX* ssl_ctx, X509* x509)
```

Comment:
This should return an int32_t, not int.  That way it is guaranteed to always be 4 bytes.  (same goes for the header declaration)